components/ModalSketch.js

```diff
@@ -0,0 +1,128 @@
+import styled from "styled-components";
+import Modal from "./Modal";
+import Image from "next/image";
+import UploadIcon from "@/public/upload.svg";
+import BinIcon from "@/public/binIcon.svg";
+import { useAtom } from "jotai";
+import { statusUploadAtom, showEditImageAtom } from "@/lib/atom";
+
+export default function ModalDelete({
+  image,
+  showModalSketch,
+  handleClose,
+  handleImageChange,
+  handleDeleteImage,
+}) {
+  const [editImage, setEditImage] = useAtom(showEditImageAtom);
+  const [statusUpload, setStatusUpload] = useAtom(statusUploadAtom);
+
+  return (
+    <Modal show={showModalSketch}>
+      <GridWrapper>
+        <TitleWrapper>
+          <h1>Your Sketch</h1>
+          <StyledButton
+            onClick={() => {
+              handleClose();
+              setEditImage(false);
+            }}
+          >
+            Close
+          </StyledButton>
+        </TitleWrapper>
+        {image.url ? (
+          <>
+            <StyledImage
+              src={image.url}
+              alt={image.alt}
+              width={image.width}
+              height={image.height}
+            />
+          </>
+        ) : (
+          <TextWrapper>
+            <p>
+              You can add a new image by clicking the <br /> <UploadIcon />
+              <br /> below
+            </p>
+          </TextWrapper>
+        )}
+        <FormWrapper>
+          <button
+            onClick={() => {
+              setEditImage(!editImage);
+            }}
+          >
+            <StyledUploadIcon />
+          </button>
+          <button onClick={handleDeleteImage}>
+            <StyledBinIcon />
+          </button>
+          {editImage ? (
+            <form onSubmit={handleImageChange}>
+              <InputWrapper>
+                <p>{statusUpload}</p>
+                <input type="file" name="imageFile" />
+                <button>upload</button>
+              </InputWrapper>
+            </form>
+          ) : null}
+        </FormWrapper>
+      </GridWrapper>
+    </Modal>
+  );
+}
+
+const TextWrapper = styled.div`
+  padding: 10em 2em;
+  text-align: center;
+`;
+
+const InputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const FormWrapper = styled.div`
+  display: flex;
+  height: 20vh;
+  flex-direction: column-reverse;
+`;
+
+const StyledUploadIcon = styled(UploadIcon)`
+  width: 36px;
+  height: 36px;
+`;
+
+const StyledBinIcon = styled(BinIcon)`
+  width: 36px;
+  height: 36px;
+`;
+
+const StyledImage = styled(Image)`
+  object-fit: contain;
+  width: 100vw;
```

Comment:
for my personal interest you are using different endings like vw ,sometimes em and vh and px. Do you have special reasons to mix them all up , or did it just work good for you like this?

Reply:
Its because it works for me 😅 but i saw it in other pr's that they use something like padding with ```em``` and ```rem```, i consider changing this in future pr's but for now i thought the pr is big enough. Also my styling is going to change completly

Reply:
> Its because it works for me 😅 but i saw it in other pr's that they use something like padding with `em` and `rem`, i consider changing this in future pr's but for now i thought the pr is big enough. Also my styling is going to change completly

thx for your answer